example/follow_me/follow_me.cpp

```diff
@@ -10,11 +10,15 @@
 * @date 2018-01-03
 */
 
+#include <chrono>
+#include <dronecore/action.h>
```

Comment:
@julianoes So are plugins now allowed to depend on other plugins?

Reply:
No, they are not. This is the example though which can depend on whatever plugins it requires.

Reply:
Thanks. Phew. Thought I was looking at a plugin.